powerlevel9k.zsh-theme

```diff
@@ -1298,6 +1298,8 @@ powerlevel9k_prepare_prompts() {
   # Reset start time
   _P9K_TIMER_START=0xFFFFFFFF
 
+  setopt PROMPT_SUBST
+
   if [[ "$POWERLEVEL9K_PROMPT_ON_NEWLINE" == true ]]; then
     PROMPT='$(print_icon 'MULTILINE_FIRST_PROMPT_PREFIX')%f%b%k$(build_left_prompt)
```

Comment:
Yeah, I guess I could be wrong though... if anything this set of PRs is telling me I need to go back and re-read a large portion of the documentation to make sure I know what I'm talking about.